src/vaadin-checkbox-group.html

```diff
@@ -348,6 +392,22 @@
             this.removeAttribute('focused');
           }
         }
+
+        /** @private */
+        _setOrToggleHasHelperAttribute() {
+          const slottedNodes = this.shadowRoot.querySelector(`[name="helper"]`).assignedNodes();
+          // Only has slotted helper if not a text node
+          // Text nodes are added by the helperText prop and not the helper slot
+          // The filter is added due to shady DOM triggering this slotchange event on helperText prop change
```

Comment:
"slotchange" listener is not in use. Hmm...why not?

Reply:
Method is being called from the `FlattenedNodesObserver` inside `ready` method.